include/cantera/zeroD/ReactorNet.h

```diff
@@ -195,6 +212,10 @@ class ReactorNet : public FuncEval
         return m_paramNames.at(p);
     }
 
+    //! Initialize the reactor network. Called automatically the first time
```

Comment:
this is now exposed in cython, as reactors need to be initialized before limits can be applied. I.e. automatic initialization happens too late.